### ISSUE TITLE
Velocity > 1

### ISSUE THREAD

**Issue by bw1984** [cotejp/webmidi#6]
Hi, i am using your excellent library to send a recieve signals from my [APC Mini](http://www.akaipro.com/product/apc-mini).

In order to debug and play around with it I have mapped all of my pads to a virtual interface of checkboxes and range sliders, and have it set up to detect the 'noteoff' event when i press any of the pads. I managed to find some basic docs for the mini which show the pad IDs which saved me a bit of time, and also noticed that i can send a signal back to the pad to tell the pad to stay lit once pressed, but in order to do this i need to be able to send a velocity signal > 1 back to it. I got it working by removing the check on line 2652 in your library but just wondered if theres a cleaner way to do it? my code to set the LEDs on the pads is provided below for clarity, and the original graphic i found which steered me in the right direction.

![APC mini mappings](https://d2r1vs3d9006ap.cloudfront.net/s3_images/1143703/apc_mini_midi.jpg?1421447412)

```javascript
output.playNote(56, 'all', {velocity:0});  // off
output.playNote(56, 'all', {velocity:1});  // green
output.playNote(56, 'all', {velocity:2});  // green_blink
output.playNote(56, 'all', {velocity:3});  // red
output.playNote(56, 'all', {velocity:4});  // red_blink
output.playNote(56, 'all', {velocity:5});  // yellow
output.playNote(56, 'all', {velocity:6});  // yellow_blink
```
any ideas?

any if anyone is interested i can upload my virtual APC mini interface, preumably it would only require slight modifications to be used with other similar devices.

**Comment by cotejp** [comment]
In MIDI, the velocity is set using an integer between `0` and `127`. However, in this library, the velocity is set using a decimal number between `0` and `1` which is usually easier to work with. This means that if you want to send an actual MIDI velocity value of `6` you would need to do the following: 
```javascript
output.playNote(56, 'all', {velocity: 0.047});
```
This would result in the value `6` being sent because 0.047 × 127 = 6 (once rounding has been applied).

Obviously, in your case, this is a bit silly and counter-intuitive. For that reason, I added an option to both `playNote()` and `stopNote()`. With that option, you will be able to send raw velocity values:
```javascript
output.playNote(56, 'all', {velocity: 6, rawVelocity: true});
```
To make it consistent, I also added a `rawVelocity` property to incoming MIDI message.

Those changes are available in release v2.0.0-beta.2 on [GitHub](https://github.com/cotejp/webmidi/releases/tag/2.0.0-beta.2) and [NPM](https://www.npmjs.com/package/webmidi).
